process_nwb/wavelet_transform.py

```diff
@@ -365,7 +362,7 @@ def store_wavelet_transform(elec_series, processing, filters='rat', hg_only=True
                                                    metric='amplitude',
                                                    source_timeseries=elec_series,
                                                    starting_time=elec_series.starting_time,
-                                                   rate=rate,
+                                                   rate=post_resample_rate,
```

Comment:
I think post_resample_rate can be `None` in this case which still isn't correct.

It might be best to remove these edits to wavelet_transform.py and I can make a PR that fixes the rate issue independently.

Reply:
Will do